src/components/stepforms/widgets/DateComponents/RelativeDateRangeForm.vue

```diff
@@ -99,21 +104,24 @@ export default class RelativeDateRangeForm extends Vue {
     this.$emit('input', { ...this.value, date: value });
   }
 
-  get directions() {
-    return [
-      { label: 'until', value: -1 },
-      { label: 'from', value: +1 },
-    ];
+  get operators() {
```

Comment:
So this is only the implemented operators, or available operators. Maybe we should name this with a prefix to mark the difference with the whole list of operators?